sklearn/inspection/_plot/decision_boundary.py

```diff
@@ -88,9 +81,28 @@ class DecisionBoundaryDisplay:
     xx1 : ndarray of shape (grid_resolution, grid_resolution)
         Second output of :func:`meshgrid <numpy.meshgrid>`.
 
-    response : ndarray of shape (grid_resolution, grid_resolution)
+    response : ndarray of shape (grid_resolution, grid_resolution) or \
+        (grid_resolution, grid_resolution, n_classes)
         Values of the response function.
 
+    multiclass_colors : list of str or str, default=None
+        Specifies how to color each class when plotting all classes of multiclass
+        problem. Ignored for binary problems and multiclass problems when plotting a
+        single prediction value per point.
+        Possible inputs:
+
+        * list, of length `response.shape[-1]`, of Matplotlib `color
```

Comment:
> Instead of response.shape[-1] can we just say n_classes or there is a corner case that I'm not thinking of?

So I've been weirdly avoiding the use of e.g., `n_classes` when the estimator does not have that attribute/parameter (because it's in backticks and 'looks' like a parameter? but honestly just neurotic-ness). But I will just use it, because I think users know what this means.